portals/e2e/features/pages/dashboard.js

```diff
@@ -5,9 +5,11 @@ function dashboard () {
   this.openAccordionMenu = openAccordionMenu;
   this.clickOnCta = clickOnCta;
   this.clickOnBrandEmulatorButton = clickOnBrandEmulatorButton;
+  this.clickOnCompanyEmulatorButton = clickOnCompanyEmulatorButton;
   this.assertLoggedIn = assertLoggedIn;
   this.logout = logout;
   this.selectBrand = selectBrand;
+  this.selectCompany = selectCompany;
```

Comment:
I have merged both `selectBrand` and `selectCompany` into `selectEmulatedEntity`